src/Core.ts

```diff
@@ -249,7 +250,7 @@ class ModuleImpl implements Module, Register {
 	}
 
 	private logError(e: RegistrationError) {
-		this.getLogger().error("", e);
```

Comment:
Switching to single quotes introduces a TSLint error

Reply:
All single quotes, as in the potential example above addressed.